components/VTheme/Profile.js

```diff
@@ -1,31 +1,56 @@
 import styled from 'styled-components'
 
 export const ProfileBanner = styled.header`
+margin: 5rem 0 0rem 0;
 display: grid;
 grid-template-columns: 100%;
 align-self: center;
 justify-self: center;
-
+margin-bottom: 2rem;
 @media screen and (min-width: 767px) {
-  grid-template-columns: 25% 75%;
+  grid-template-columns: 15rem 1fr;
+  gap: 5rem;
+ 
 }
 
 img {
-  width: 16rem;
-  max-height: 12rem;
+  width: 20rem;
```

Comment:
If we expect more than one image inside a banner this will affect both of them. Using the VBannerImage described below will distinguish it from any others.  This section may be unnecessarily duplicating styles.

Reply:
There aren't any cases where there's more than one image though

either way, this has been refactored out